Makefile

```diff
@@ -99,8 +97,8 @@ $(BINDIR)/deepcopy-gen: cmd/libs/go2idl/deepcopy-gen
 	$(DOCKER) glide install --strip-vendor
 	touch $@
 
-.scBuildImage: hack/Dockerfile
-	sed "s/GO_VERSION/$(GO_VERSION)/g" < hack/Dockerfile | \
+.scBuildImage: build/build-image/Dockerfile
+	sed "s/GO_VERSION/$(GO_VERSION)/g" < build/build-image/Dockerfile | \
```

Comment:
The name of the game here is re-organizing to incrementally advance our repo in similarity to k8s. This is where they keep the Dockerfile for their development environment, so I took my cues from that.

See path to the Dockerfile in the link below:

https://github.com/kubernetes/kubernetes/blob/master/build/build-image/Dockerfile